### PR TITLE
Align java scripts to the default UI (if possible) + a small css change

### PR DESCRIPTION
**Pull request by mmattel**
This is the next step in aligning our UI to the [Antora default UI](https://gitlab.com/mmattel/antora-ui-default) with regards to rules and scripts.

The intention is to make a working base for our UI while keeping the layout but to be able to update it with stuff coming from the default one. In the current situation we disalign so much that upadates for scripts or css cant be incorporated easily nor we can check for changed stuff without a lot of extra work.

This PR:
* updates nearly all scripts to be aligned with the default UI.
* *does not* update the `01-navigation.js` file as the new version from the default UI breakes things.
* *does not* update the `relativize.js` file as the new version from the default UI breakes things.
* adapts the linter rules to enable the changes.
* the numbering of the scripts are also aligned to the default UI making it easier for identification.
* tabs.css got a small update
* tried to move some css :root variable definitions into a global vars.css file as used in the default UI, so we can more easily add/adapt/change css stuff.
BUT this does not work as some variables with the same name are defined in multiple css files with different values. This needs special fixing.
* as preperation, an empty vars.css file onyl containing notes has been generated to ease fixing stuff the next upcoming round

Made with 😅 a lot of 🧡 and tested until my head was 🤪

### CODE REVIEW

**Comment by mmattel** [comment]
Found a glitch, fixing...

**Comment by mmattel** [comment]
The glitch is fixed, rending is fine, content has no breakes 😄

**Comment by mmattel** [comment]
Just rebasing...

File: src/js/01-navigation.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
-(function () {
+/* do not use the content of the script from the default ui */
+/* https://gitlab.com/mmattel/antora-ui-default/-/tree/master/src/js */
+/* it curretly breakes the navigation (no clicking possible) most likely */
```
Comment (phil-davis) [inline src/js/01-navigation.js:3]:
```suggestion
/* it currently breaks the navigation (no clicking possible) most likely */
```

File: src/js/01-navigation.js [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+/* because the referenced names are different in css */
+/* this needs an in depth investigation and update including a lot of testing */
+/* to avoid any accidentally updates, the script has a differernt name compared to the default ui*/
```
Comment (phil-davis) [inline src/js/01-navigation.js:6]:
```suggestion
/* to avoid any accidental updates, the script has a different name compared to the default ui*/
```

**Review by phil-davis — COMMENTED** [review]
I am not the one to review the technical content of this.

**Review by EParzefall — APPROVED** [review]

**Pull request merged**